test/Microsoft.AspNet.Mvc.Core.Test/ApplicationModel/DefaultActionModelBuilderTest.cs

```diff
@@ -646,6 +646,22 @@ public void GetActions_MixedHttpVerbsAndRoutes_MultipleEmptyAndNonEmptyVerbs()
             Assert.Equal<string>(new string[] { "GET" }, action.HttpMethods);
         }
 
+        [Fact]
```

Comment:
What happens in uber binding in the following two scenarios
1. A property has two attributes
2. (probably more of an issue) derived property has an attribute and also the property on the base has an attribute